.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,12 +5,12 @@
 
 **How did you make sure your solution works?**
 
-**Did you add any tests?**
-
 **Are there any special changes in the code that we should be aware of?**
 
-**Did you run `make lint` and `make test`?**
-
-**Are you making a PR to a feature branch or development rather than master?**
+**Please check the following, if applicable:**
 
-**Did you add an entry to `CHANGELOG.rst`? (if not, please do)**
+- [ ] Did you add any tests?
+- [ ] Did you run `make lint`?
+- [ ] Did you run `make test`?
+- [ ] Are you making a PR to a feature branch or development rather than master?
```

Comment:
this question cannot be handled by a check box

Reply:
yes, it can, it still is a yes/no question.

Reply:
After reading it again I guess we do not differentiate between Dev vs feature?

Reply:
ok nm